deployment/ansible/roles/driver.app/tasks/main.yml

```diff
@@ -51,6 +51,5 @@
 - name: Run Django migrations
   command: >
     /usr/bin/docker exec -i driver-app ./manage.py migrate
-  when: developing or staging
```

Comment:
Yeah, the main problem here is that without running the migrations, performing a fresh deploy will fail when attempting to add Windshaft access roles (since the tables references by the role don't exist). So at a minimum, we'd need to ensure migrations are run automatically at least based on some condition (the absence of those tables?). But regardless, I think running them automatically here will be what must users will desire in the general case. Definitely do need to keep it in mind for future migrations though.